src/modules/person/mapper/person-api.mapper.ts

```diff
@@ -36,31 +47,31 @@ export class PersonApiMapper {
                 revision: person.revision,
                 personalnummer: person.personalnummer,
             },
-            personenkontexte: kontexte.map((kontext: PersonenkontextDo<true>) =>
-                this.mapToPersonenkontextResponse(kontext),
-            ),
+            personenkontexte: personenkontexte,
             gruppen: [], // TODO: if the gruppe module is implemented, this should be filled out with EW-656 / EW-697
         });
 
         return response;
     }
 
-    private mapToPersonenkontextResponse(kontext: PersonenkontextDo<true>): PersonenkontextResponse {
-        const response: PersonenkontextResponse = PersonenkontextResponse.new({
-            id: kontext.id,
-            referrer: kontext.referrer,
-            mandant: kontext.mandant,
-            organisation: {
-                id: kontext.organisationId,
-            },
-            rolle: kontext.rolle,
-            personenstatus: kontext.personenstatus,
-            jahrgangsstufe: kontext.jahrgangsstufe,
-            sichtfreigabe: kontext.sichtfreigabe,
-            loeschung: kontext.loeschungZeitpunkt ? { zeitpunkt: kontext.loeschungZeitpunkt } : undefined,
-            revision: kontext.revision,
+    public async mapToPersonenkontextResponse(props: Personenkontext<true>): Promise<PersonenkontextResponse> {
+        const rolle: Option<Rolle<true>> = await props.getRolle();
+        const response: PersonenkontextResponse = new PersonenkontextResponse({
+            id: props.id,
+            referrer: props.referrer,
+            mandant: props.mandant!,
+            organisation: CreatedPersonenkontextOrganisation.new({ id: props.organisationId }),
+            personenstatus: props.personenstatus,
+            jahrgangsstufe: props.jahrgangsstufe,
+            sichtfreigabe: props.sichtfreigabe,
+            loeschung: props.loeschungZeitpunkt
+                ? LoeschungResponse.new({ zeitpunkt: props.loeschungZeitpunkt })
+                : undefined,
+            revision: props.revision,
         });
 
+        response.roleName = rolle ? rolle.name : undefined;
+
         return response;
     }
```

Comment:
We cannot not add it a static in the method in the PersonenkontextResponse. It would not work, because we are using automapper for PersonenkontextResponse, if we do that we gonna have to setup up this method in automapper and also add Personenkontext aggregate in the mapping profile, which means map all the properties of Personenkontext with automapper, and we do not want to go that direction.